### DISCUSSION TITLE
View quote posts from original post

### DISCUSSION THREAD

**Discussion by thieflord06** [bluesky-social/atproto#1584] [Bluesky Client App]
When you click on the number of repost that shows you who reposted a post, it should also show the quote posts and its contents too. Or there should be a separate item to click to view quote posts.

**Comment by mackuba** [comment]
+1 - I would just add another field next to likes and reposts like this:

<img width="618" alt="Screen Shot 2023-09-24 at 22 03 46" src="https://github.com/bluesky-social/atproto/assets/28465/48ea313c-aae3-45ac-be66-c75daa1aba23">

And then when you click it, it would show a list in the same way as likes and reposts currently do:

<img width="676" alt="Screen Shot 2023-09-24 at 22 07 38" src="https://github.com/bluesky-social/atproto/assets/28465/635611e2-0d89-4e50-9de9-c8b42f70914c">

Also we'd probably need a matching `app.bsky.feed.getQuotes` endpoint in the API like `app.bsky.feed.getRepostedBy` and `app.bsky.feed.getLikes`.

**Reply by mackuba** [reply]
Twitter has recently removed the quotes count label/button and moved it to the popup menu when you click "repost", but I don't think that makes sense since there's plenty of space in that row:

<img width="410" alt="Screen Shot 2023-09-24 at 22 14 12" src="https://github.com/bluesky-social/atproto/assets/28465/0d3467fa-04f7-499c-bb3d-267e5dc6fd5a">

**Reply by ShreyanJain9** [reply]
I would prefer, instead of a list of profiles, directly showing you the posts that quote it

**Reply by mackuba** [reply]
@ShreyanJain9 ahh yes, of course 🙈

So something like that:

<img width="606" alt="Screen Shot 2023-09-25 at 20 24 12" src="https://github.com/bluesky-social/atproto/assets/28465/d698480a-6f52-484c-b640-d7dfde675691">

**Reply by venteto** [reply]
I agree, there's multiple times I visit a post via a quote post, but once on the original post, there is no hint at all of the quote post. They should be listed too.

I prefer the idea of a _separate_ icon for quote posts (rather than hiding‡ it behind an additional click to expand a repost _types_ menu), _between_ the basic repost and likes icon. I could just simply be another repost icon but prefixed or surrounded by quotes. Or it could be just a pair of guillemets (better quotation marks) «».

‡ anything that can be done to reduce the number of clicks needed to accomplish something in the UI is preferable. For example, I don't care how long the left column menu is, it could have 50 items (in both iconified and expanded column widths) in it for all I care, and that shouldn't make it any less compatible with a responsive small screen layout either (but please don't make us desktop users suffer simply because small screen users exist!). Scrolling is not a problem, additional clicks are a problem (which is why limiting the right column list of pinned feeds to something like 26 serves no purpose, let us have infinite pinned feeds / user lists in that column).

**Comment by rudyfraser** [comment]
This is being brought up again by a few users in the Blacksky feed.
[https://bsky.app/profile/gothamgirlblue.com/post/3krzthzguwo26](url)

**Reply by surfdude29** [reply]
The link seems to be broken, this one works:

https://bsky.app/profile/gothamgirlblue.com/post/3krzthzguwo26

**Comment by cooljeanius** [comment]
I remember on Twitter, before they implemented this feature, there used to be a bot that you could tag to get it to send you the URL for a search that would find all posts quoting the replied-to post; perhaps someone could implement something similar for BlueSky while we wait for native support to be added?

**Reply by mackuba** [reply]
Well, I made this thing 😉 https://blue.mackuba.eu/skythread/?quotes=https://bsky.app/profile/jay.bsky.team/post/3krwtkge5rl2t

If you go to https://blue.mackuba.eu/skythread/ and paste a link to a post to the search bar, you will see a link e.g. "14 quotes" below the post in the status line with the like/RT count, if it has any quotes (if there are no quotes then this doesn't appear).

**Reply by cooljeanius** [reply]
Cool, is there a bot that we can mention that will automatically reply with the relevant link?
